wren-modeling-rs/src/mdl/builder.rs

```diff
@@ -0,0 +1,487 @@
+#![allow(dead_code)]
+
+use crate::mdl::manifest::{
+    Column, JoinType, Manifest, Metric, Model, Relationship, TimeGrain, TimeUnit, View,
+};
+use std::sync::Arc;
+
+/// A builder for creating a Manifest
+pub struct ManifestBuilder {
+    pub manifest: Manifest,
+}
+
+impl ManifestBuilder {
+    pub fn new() -> Self {
+        Self {
+            manifest: Manifest {
+                catalog: "wrenai".to_string(),
+                schema: "default".to_string(),
+                models: vec![],
+                relationships: vec![],
+                metrics: vec![],
+                views: vec![],
+            },
+        }
+    }
+
+    pub fn with_catalog(mut self, catalog: &str) -> Self {
```

Comment:
How about just `catalog`?

Reply:
Just `catalog` may be confused with `get_catalog`. I think having a `with` prefix is better.